compatibility/bazel_tools/create-daml-app/index.test.ts

```diff
@@ -138,7 +138,9 @@ beforeAll(async () => {
 
   const sandboxOnXOptions = [
     `--ledger-id=${SANDBOX_LEDGER_ID}`,
-    `--participant=participant-id=sandbox,port=0,port-file=${SANDBOX_PORT_FILE_NAME}`
+    `--participant=participant-id=sandbox,port=0,port-file=${SANDBOX_PORT_FILE_NAME}`,
+    // TODO https://github.com/digital-asset/daml/issues/12932
+    `--disable-conflict-checking`,
```

Comment:
```suggestion
```
fixed in #12936

Reply:
Ok, will replace

Reply:
Thanks @moritzkiefer-da . Done e92e52ff05475f1cbf571504c51bc4e398f1e175